alchemist-web-renderer/build.gradle.kts

```
@@ -90,35 +89,24 @@ application {
 }
 
 tasks.getByName<JavaExec>("run") {
-    classpath(tasks.getByName<Jar>("shadowJar"))
+    val shadowJarTask = tasks.getByName<Jar>("shadowJar")
+    dependsOn(shadowJarTask)
+    classpath(shadowJarTask)
 }
 
 /**
- * Configure a [com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar] task named "jvmShadowJar" for the jvm target
- * that also includes the js artifacts.
+ * Configure the [ShadowJar] task to work exactly like the "jvmJar" task of Kotlin Multiplatform, but also
+ * include the JS artifacts by depending on the "jsBrowserProductionWebpack" task.
  */
-fun registerShadowJar() {
-    kotlin.targets.named<KotlinJvmTarget>("jvm") {
-        compilations.named("main") {
-            val webpackTask = tasks.getByName<KotlinWebpack>("jsBrowserProductionWebpack")
-            val shadowJar = tasks.register<ShadowJar>("jvmShadowJar") {
-                dependsOn(webpackTask)
-                from(File(webpackTask.destinationDirectory, webpackTask.outputFileName))
-                from(output)
-                configurations = listOf(runtimeDependencyFiles)
-                archiveClassifier.set("all")
-            }
-            listOf("jvmJar", "shadowJar").forEach {
-                tasks.named<Jar>(it) {
-                    finalizedBy(shadowJar)
-                }
-            }
-        }
-    }
+tasks.withType<ShadowJar> {
```

configuration avoidance
```suggestion
tasks.withType<ShadowJar>().configureEach {
```